src/geo/projection/globe.js

```diff
@@ -288,11 +288,9 @@ export function globeBuffersForTileMesh(painter: Painter, tile: Tile, coord: Ove
     return [gridBuffer, poleBuffer];
 }
 
-export function globeMatrixForTile(id: CanonicalTileID, globeMatrix: mat4) {
+export function globeMatrixForTile(id: CanonicalTileID, globeMatrix: Float64Array) {
```

Comment:
Not related to this PR, but I'll open a ticket to annotate return values of these functions to make these stricter with regards to flow.

Reply:
As we upgrade to newer versions of Flow, this will be required anyway — all exported functions and methods will need to be fully annotated for the new typing architecture to work.